linode/manager
# test: [M3-8526] - Update remaining Linode Create Cypress tests run against Linode Create v2

## Description 📝

- Updates remaining Cypress test to run against Linode Create v2 rather than Linode Create v1 🧪 
- This PR will allow be to begin deleting Linode Create v1

## How to test 🧪

- Verify Cypress tests pass ✅ 
- Verify **all** Linode Create related tests run against Linode Create v2

## As an Author I have considered 🤔
